src/LiveComponent/assets/src/live_controller.ts

```diff
@@ -296,11 +296,24 @@ export default class extends Controller {
         }
     }
 
-    _makeRequest(action: string|null) {
+    _makeRequest(action: string|null, values: Array<string>|object) {
```

Comment:
Prefer use `Record<K,V>` instead of `object`:
```suggestion
    _makeRequest(action: string|null, values: Array<string>|Record<string,unknown>) {
```

Reply:
Sure. I'm not familiar with TS :)